openssl/openssl.go

```diff
@@ -35,21 +42,74 @@ func Init() error {
 	return nil
 }
 
+// providerAvailable looks through provider's digests
+// checking if there is any that matches the pprog query.
+func providerAvailable(pprog *C.char) bool {
```

Comment:
I'm not sure what `pprog` means here, my guess would be "property program"? The `EVP_MD_fetch` doc seems to just call it `properties`.

Below there's `pprop` which seems like "provider properites", maybe just a typo?

Reply:
I meant to use `prop` instead of `pprop`, and then that typo propagated through the codebase 😝 

`prop` is  used by OpenSSL in several places, and stands for property query string. I will rename all `pprop` occurrences to `props`, which is more commonly understood that `prop`.